### PR TITLE
Semantics usage: support for Hebrew and Arabic

### PR DESCRIPTION
**Pull request by yanivshaked**

### CODE REVIEW

**Comment by yanivshaked** [comment]
Related to #239

File: lib/src/delegates/asset_picker_text_delegate.dart [hunk 1]
```diff
@@ -545,6 +579,39 @@ class ArabicAssetPickerTextDelegate extends AssetPickerTextDelegate {
...
+  String get sTypeImageLabel => 'صورة';
+
+  @override
+  String get sTypeVideoLabel => 'فيديو';
+
+  @override
+  String get sTypeOtherLabel => 'آخر';
+
+  @override
+  String get sActionPlayHint => 'العب';
```
Comment (maxzod) [inline lib/src/delegates/asset_picker_text_delegate.dart:596]: use `تشغيل`  
`العب` 
means play but in the context of  games not audio
image

File: lib/src/delegates/asset_picker_text_delegate.dart [hunk 2]
```diff
@@ -545,6 +579,39 @@ class ArabicAssetPickerTextDelegate extends AssetPickerTextDelegate {
...
+
+  @override
+  String get sActionPreviewHint => 'معاينة';
+
+  @override
+  String get sActionSelectHint => 'تحديد';
+
+  @override
+  String get sActionSwitchPathLabel => 'مسار التبديل';
```
Comment (maxzod) [inline lib/src/delegates/asset_picker_text_delegate.dart:605]: `تبديل المسار `

**Review by maxzod — CHANGES_REQUESTED** [review]
Great work  , i have two words to change

**Comment by yanivshaked** [comment]
Thanks! updated according to your comments

**Review by maxzod — APPROVED** [review]

**Pull request merged**